### PR TITLE
Add weakly covers symbol and opposites shortcuts

### PR DESCRIPTION
**Pull request by YaelDillies**
Relevant mathlib PRs:
* leanprover-community/mathlib#13424 for [`⩿`](https://leanprover-community.github.io/mathlib_docs/order/cover.html#wcovby)
* leanprover-community/mathlib#13798 for [`ᵒᵈ`](https://leanprover-community.github.io/mathlib_docs/order/basic.html#order_dual)

### CODE REVIEW

File: src/abbreviation/abbreviations.json [hunk 1]
```diff
@@ -442,6 +445,9 @@
     "simplefunc": "→ₛ",
 
     "covers": "⋖",
+    "covers": "⋖",
```
Comment (vihdzp) [inline src/abbreviation/abbreviations.json:448]: This is a duplicate.

Comment (YaelDillies) [reply src/abbreviation/abbreviations.json:448]:
```suggestion
    "covby": "⋖",
```

**Comment by alreadydone** [comment]
Should we add `\<|` (and possibly `\lf`) for `⧏` here or in another PR?

**Comment by gebner** [comment]
> Should we add \<| (and possibly \lf) for ⧏ here or in another PR?

Please first discuss on Zulip, and then a new PR please.

**Pull request merged**